Documentation/ApiOverview/DirectoryStructure/Index.rst

```diff
@@ -21,96 +21,127 @@ almost always find the directory structure as outlined below.
-      further storage locations configured outside of :file:`fileadmin/`, even
-      pointing to different servers or using 3rd party digital asset management
-      systems.
-
-      .. note::
-        Note this directory is meant for editors! Integrators should
-        *not* locate frontend website layout related files in here: Storing
-        HTML templates, logos, Css and similar files used to build the website
-        layout in here is considered bad practice. Integrators should locate
-        and ship these files within a project specific extension.
-
-  - :Directory: :file:`typo3/`
-    :Description:
-      TYPO3 Backend directory. This directory contains most of the files
-      coming with the TYPO3 Core. The files are arranged logically in the
-      different system extensions in the :file:`sysext/` directory,
-      according to the application area of the particular file. For example,
-      the ":code:`frontend`" extension amongst other things contains the
-      "TypoScript library", the code for generating the Frontend website. In
-      each system extension the PHP files are located in the folder
-      :file:`Classes/`. See :ref:`extension files locations <extension-files-locations>`
-      for more information on how single extensions are structured.
-
-  - :Directory: :ref:`Environment-config-path` either :file:`typo3conf/` or :file:`config/`
-    :Description:
-      TYPO3 configuration directory. This directory contains installation wide
-      configuration.
-
-      Amongst others, this directory contains the files :file:`LocalConfiguration.php` and
-      :file:`AdditionalConfiguration.php`. See chapter
-      :ref:`Configuration files <configuration-files>` for details.
-
-  - :Directory: :file:`typo3conf/ext/`
-    :Description:
-      Directory for local TYPO3 extensions. Each subdirectory contains one extension.
-
-  - :Directory: :ref:`Environment-labels-path` either :file:`typo3conf/l10n` or :file:`var/labels`
-    :Description:
-      Directory for extension localisations. Contains all downloaded translation
-      files.
-
-  - :Directory: :file:`typo3temp/`
-    :Description:
-      Directory for temporary files. It contains subdirectories (see below)
-      for temporary files of extensions and TYPO3 components.
-
-  - :Directory: :file:`typo3temp/assets/`
-    :Description:
-      Directory for temporary files that should be public available
-      (e.g. generated images).
-
-  - :Directory: :file:`typo3temp/var/`
-    :Description:
-      Directory for temporary files that contains private files (e.g. cached
-      Fluid templates) and should not be publicly available.
-      See also :ref:`Environment-configuring-paths` for a more detailed description.
+Directories in a typical project
+================================
+
+.. contents::
+   :local:
+
+
+:file:`public/`
+---------------
+
+:file:`public/_assets/`
+~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
```suggestion
:file:`public/`
---------------

We assume here that your web root points to a folder called :file:`public` in a Composer-based installation or a legacy 
installation as is commonly done. Otherwise replace  :file:`public` with path to your web root.

:file:`public/_assets/`
~~~~~~~~~~~~~~~~~~~~~~~
```